NAMESPACE

```diff
@@ -1,3 +1,16 @@
 # Generated by roxygen2: do not edit by hand
 
+export(Font)

```

Comment:
@pchelle Time to use the package development
document is created using devtools::document() (CTRL+ALT+D)
package is loaded using CTR+ALT+L

Before submitting code: Run package styling
Run check: CTRL+ALT+E. It will show all erros with the package